views/games-events.html

```diff
@@ -41,6 +41,12 @@ <h2>Games and Events for {{calendarTitle}}</h2>
                           cell-auto-open-disabled="true" on-timespan-click="timespanClicked(calendarDate, calendarCell)">
             </mwl-calendar>
             <br>
+            <div>
+                <a class="btn btn-default pull-right" href="https://calendar.google.com/calendar/ical/rpiambulance.com_uuc6tmlmf4ct8qeps19oj1i69g%40group.calendar.google.com/public/basic.ics">
```

Comment:
Is it possible to make this secret? I can reset the link and provide another one to you.

Reply:
This is also public information and therefore doesn't need to be hidden. The less things that we have to remove from the github the better.